code/modules/security_levels/keycard_authentication.dm

```diff
@@ -165,7 +165,7 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/keycard_auth/wall_mounted, 26)
 /obj/machinery/keycard_auth/wall_mounted/Initialize(mapload)
 	. = ..()
 	if(mapload)
-		find_and_hang_on_atom()
+		find_and_mount_on_atom()
```

Comment:
As mentioned previously `/obj/machinery/keycard_auth` (the base type) is intended to mount to tables